lib/modules/dosomething/dosomething_rogue/dosomething_rogue.module

```diff
@@ -66,7 +66,7 @@ function dosomething_rogue_send_reportback_to_rogue($values, $user = NULL, $fail
   // from northstar directly.
   if (!$northstar_id) {
     $northstar_user = dosomething_northstar_get_user($user->uid, 'drupal_id');
-   $northstar_id = $northstar_user->id;
+    $northstar_id = $northstar_user->id;
```

Comment:
it's weird cause a lot of places you clean up the extra spaces, but here it added one...